app/team.go

```diff
@@ -685,11 +684,11 @@ func (a *App) GetTeamsForUser(userId string) ([]*model.Team, *model.AppError) {
 }
 
 func (a *App) GetTeamMember(teamId, userId string) (*model.TeamMember, *model.AppError) {
-	result := <-a.Srv.Store.Team().GetMember(teamId, userId)
```

Comment:
Here you can just `return a.Srv.Store.Team().GetMember(teamId, userId)`

Reply:
You are right!